lib/web/ui/cluster.go

```diff
@@ -46,7 +46,7 @@ type Cluster struct {
 	ProxyVersion string `json:"proxyVersion"`
 }
 
-//nolint:unused,deadcode
+//nolint:unused,deadcode,varcheck
```

Comment:
If this is dead code, let's just remove it.

Reply:
Done.
Wasn't sure what the policy on these global loggers was.